tasty/source_shapes/haystack/nrel.json

```diff
@@ -28,6 +145,46 @@
           }
         ]
       }
+    },
+    {
+      "name": "NREL-VAV-SD-Cooling-Only-Shape",
+      "types": ["coolingOnly-vav"],
+      "predicates": {
+        "requires": [
```

Comment:
check back to the spreadsheet for what is required vs optional (column I)

Reply:
👍